### PR TITLE
Add FileManager.list_files_from_disk

### PR DESCRIPTION
**Pull request by vasinov**

### CODE REVIEW

**Review by collindutter — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+- Added `FileManager.list_files_from_disk`
```
Comment (collindutter) [inline CHANGELOG.md:10]: Nit: `Added` is redundant. Maybe reword to "- `list_files_from_disk` activity to `FileManager` Tool."

**Review by collindutter — APPROVED** [review]

**Review by dylanholmes — APPROVED** [review]
